bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -84,13 +79,15 @@ else if (!isRegistered(userService.getCurrentUser().getUserId()) && !request.get
 }
 
   public boolean isRegistered(String id) {
```

Comment:
Nit: name this method "userIsRegistered"

Reply:
Renamed